apps/src/lib/ui/accounts/DeleteAccountHelpers.jsx

```diff
@@ -52,50 +57,39 @@ const CHECKBOX_MAP = {
     checked: false,
     label: (
       <span>
-        <strong>{i18n.deleteAccountDialog_checkbox1_1()}</strong>
-        {i18n.deleteAccountDialog_checkbox1_2()}
-        <a href={RELEASE_OR_DELETE_RECORDS_EXPLANATION} target="_blank">
-          {i18n.learnMore()}
-        </a>
+        {renderedMarkdown(i18n.deleteAccountDialog_checkbox1())}
```

Comment:
i like that you pulled this functionality into a helper method! although in this case, because you're passing through a single param as a prop, i think you could keep `<UnsafeRenderedMarkdown markdown={i18n.deleteAccountDialog_checkbox1()} />` here and in the labels below